django/contrib/admin/options.py

```diff
@@ -557,14 +573,33 @@ def wrapper(*args, **kwargs):
         urlpatterns = [
             url(r'^$', wrap(self.changelist_view), name='%s_%s_changelist' % info),
             url(r'^add/$', wrap(self.add_view), name='%s_%s_add' % info),
+            url(r'^autocomplete/(?P<field_name>.+)/$',
+                wrap(AutocompleteJsonView.as_view(
+                    admin_site=self.admin_site,
+                    source_model=self.model,
+                    source_model_admin=self,
+                )), name='%s_%s_autocomplete' % info),
             url(r'^(.+)/history/$', wrap(self.history_view), name='%s_%s_history' % info),
             url(r'^(.+)/delete/$', wrap(self.delete_view), name='%s_%s_delete' % info),
             url(r'^(.+)/change/$', wrap(self.change_view), name='%s_%s_change' % info),
+        ]
+        for inline in self.inlines:
```

Comment:
I assume this isn't going to work with subclasses that override `get_inline_instances()`? Not sure if it's worth it, just mentioning it.